content/swiftui/concepts/viewmodifier/terms/fontWeight/fontWeight.md

```diff
@@ -0,0 +1,56 @@
+---
+Title: '.fontWeight()'
+Description: 'Sets the font weight of the text in Text view.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Modifiers'
+  - 'Xcode'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.fontWeight()`** modifier sets the font weight. We can apply the following weights:
+
+- ultraLight
+- thin
+- light
+- regular
+- medium
+- semibold
+- bold
+- heavy
+- black
+
+## Syntax
+
+```pseudo
+Text("your ultraLight text")
+  .fontWeight(.ultraLight)
```

Comment:
```suggestion
  .fontWeight(.weight)
```

I'd like the syntax example to be more general. Then in the following description you can say "`.weight` can be one of the following values" indicating the bulleted list that we're moving after the code block.